op-chain-ops/state/memory_db.go

```diff
@@ -312,3 +313,18 @@ func (db *MemoryStateDB) ForEachStorage(addr common.Address, cb func(common.Hash
 	}
 	return nil
 }
+
+func (db *MemoryStateDB) GetTransientState(addr common.Address, key common.Hash) common.Hash {
+	//TODO implement me
+	panic("implement me")
+}
+
+func (db *MemoryStateDB) SetTransientState(addr common.Address, key, value common.Hash) {
+	//TODO implement me
+	panic("implement me")
+}
+
+func (db *MemoryStateDB) Prepare(rules params.Rules, sender, coinbase common.Address, dest *common.Address, precompiles []common.Address, txAccesses types.AccessList) {
+	//TODO implement me
+	panic("implement me")
+}
```

Comment:
Should we remove the TODOs here? Looks like there's a number of functions we deliberately don't implement in this version so can just improve the panic messages and leave it like that.

Reply:
@tynes you introduced this transient state functionality in upstream geth right? Do you think we need to implement these functions?

Reply:
I did implement this in upstream, I think its fine to leave them unimplemented because they will not be called by us

Reply:
Perhaps `panic("Prepare unimplemented")` etc